src/north/filewriter/FileWriter.class.spec.js

```diff
@@ -1,7 +1,11 @@
 const FileWriter = require('./FileWriter.class')
```

Comment:
Please remove this test file from your pr and add a specific pr for it as we discussed

Reply:
Okay, I will do it , sorry i didn't commit at the right place